functions/src/modules/pm/groups/getGroup.ts

```diff
@@ -1,5 +1,25 @@
-import { ExpressFunction } from "../../../@types";
+import type { ExpressFunction } from "../../../@types";
+import { firebaseApp } from "../../../config/firebaseConfig";
+import { expressErrorHandlerFactory } from "../../../helpers";
 
-const replaceMe: ExpressFunction = (req, res, next) => {};
+const getGroup: ExpressFunction = (req, res, next) => { 
+   const errorHandler = expressErrorHandlerFactory(req, res, next);
+    firebaseApp
+    .firestore()
+    .collection("PrizeCategoryGroup")
```

Comment:
Can we call this `prizeGroups` to stick with the naming convention from other modules?
```suggestion
    .collection("prizeGroups")
```